exasol_integration_test_docker_environment/lib/test_environment/analyze_test_container.py

```diff
@@ -83,7 +88,7 @@ def get_goals(self):
 
 class DockerTestContainerBuild(DockerTestContainerBuildBase):
 
-    def run_task(self):
+    def run_task(self) -> Iterator[DockerCreateImageTask]:
```

Comment:
`DockerCreateImageTask` is not 100% correct, as it can be different classes. Maybe in another PR...